tests/test_core/test_loggable.py

```diff
@@ -1,5 +1,4 @@
 import unittest
-from mock import patch
```

Comment:
Don't we still need this?

Reply:
Oops a bit over zealous in my deleting after I'd added some more imports from mock I didn't need. Yes we still need it, so have put back in